main.go

```diff
@@ -537,19 +537,38 @@ func main() {
 	}
 
 	// Simulator infos
-	simulator, err := xcodeutil.GetSimulator(configs.SimulatorPlatform, configs.SimulatorDevice, configs.SimulatorOsVersion)
-	if err != nil {
+	var simulator simulatorPkg.InfoModel
+	var errGetSimulator error
+	if configs.SimulatorOsVersion == "latest" {
+		simulatorPlatformSplit := strings.Split(configs.SimulatorPlatform, " Simulator")
+		if len(simulatorPlatformSplit) == 0 {
+			errGetSimulator = fmt.Errorf("failed to parse simulator platform (%s)", configs.SimulatorPlatform)
+		}
+
+		var simulatorDevice = configs.SimulatorDevice
+		if simulatorDevice == "iPad" {
+			log.Warnf("Given device (%s) is deprecated, using (iPad 2)...", simulatorDevice)
+			simulatorDevice = "iPad 2"
+		}
+
+		desiredPlatform := simulatorPlatformSplit[0]
+		simulator, _, errGetSimulator = simulatorPkg.GetLatestSimulatorInfoAndVersion(desiredPlatform, simulatorDevice)
+	} else {
+		simulator, errGetSimulator = simulatorPkg.GetSimulatorInfo(configs.SimulatorOsVersion, configs.SimulatorDevice)
+	}
+
+	if errGetSimulator != nil {
 		if err := cmd.ExportEnvironmentWithEnvman("BITRISE_XCODE_TEST_RESULT", "failed"); err != nil {
 			log.Warnf("Failed to export: BITRISE_XCODE_TEST_RESULT, error: %s", err)
 		}
-		fail("failed to get simulator udid, error: ", err)
+		fail("failed to get simulator udid, error: ", errGetSimulator)
 	}
 
 	log.Infof("Simulator infos")
-	log.Printf("* simulator_name: %s, UDID: %s, status: %s", simulator.Name, simulator.SimID, simulator.Status)
+	log.Printf("* simulator_name: %s, UDID: %s, status: %s", simulator.Name, simulator.ID, simulator.Status)
```

Comment:
It's not in the ticket, but could you please write the iOS version too?
Because if the `SimulatorOsVersion` = Latest we have no idea about the version :(

![Screenshot 2019-10-14 at 13 45 40](https://user-images.githubusercontent.com/37296013/66748999-f0893e80-ee88-11e9-9e93-e42a148de9b5.png)